Foundation/NSTask.swift

```diff
@@ -7,6 +7,7 @@
 // See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 //
 
+#if !os(Android) // not available
```

Comment:
Still not happy about this.

Is there a way, instead, to mark the class as unavailable on the Android platform?